Source/WebCore/platform/ContextMenuItem.cpp

```diff
@@ -197,6 +197,11 @@ bool isValidContextMenuAction(ContextMenuAction action)
     case ContextMenuAction::ContextMenuItemPDFZoomOut:
     case ContextMenuAction::ContextMenuItemPDFAutoSize:
     case ContextMenuAction::ContextMenuItemPDFSinglePage:
+#if ENABLE(PDFJS)
+    case ContextMenuAction::ContextMenuItemPDFSinglePageContinuous:
+    case ContextMenuAction::ContextMenuItemPDFTwoPages:
+    case ContextMenuAction::ContextMenuItemPDFTwoPagesContinuous:
+#endif
```

Comment:
I would remove the build flag around the cases, since they are defined everywhere, and instead, return `true` or `false` depending on its value:

```
case ContextMenuAction::ContextMenuItemPDFSinglePageContinuous:
case ContextMenuAction::ContextMenuItemPDFTwoPages:
case ContextMenuAction::ContextMenuItemPDFTwoPagesContinuous: {
#if ENABLE(PDFJS)
    return true;
#else
    return false;
#endif
}
```